cvat/apps/engine/static/engine/js/shapeCreator.js

```diff
@@ -517,34 +533,27 @@ class ShapeCreatorView {
             this._mode = model.defaultMode;
 
             if (!['polygon', 'polyline', 'points'].includes(this._type)) {
+                if (!model._usingShortkey) {
+                    this._aimCoord = {
+                        x: 0,
+                        y: 0
+                    };
+                }
                 this._drawAim();
-                this._playerFrame.on('mousemove.shapeCreatorAIM', (e) => {
-                    this._aimCoord = window.cvat.translate.point.clientToCanvas(this._frameContent.node, e.clientX, e.clientY);
-                    if (this._aim) {
-                        this._aim.x.attr({
-                            y1: this._aimCoord.y,
-                            y2: this._aimCoord.y,
-                        });
-
-                        this._aim.y.attr({
-                            x1: this._aimCoord.x,
-                            x2: this._aimCoord.x,
-                        });
-                    }
-                });
             }
 
             this._createButton.text("Stop Creation");
             document.oncontextmenu = () => false;
-            this._create();
+            this._create(model._usingShortkey);
+            if (!model._usingShortkey) {
```

Comment:
Why do you need this block of code?
There is exactly the same code a couple of lines above?